src/Invoice_GenUI/Invoice_GenUI/ViewModels/CreateClientViewModel.cs

```diff
@@ -42,13 +42,13 @@ public CreateClientViewModel(INavigationService navService, IClientService clien
         [RelayCommand]
         private void GoBack()
         {
-            Navigation.NavigateTo<HomeViewModel>();
+            _navigation.NavigateTo<HomeViewModel>();
         }
         [RelayCommand]
         private async Task CreateClient()
         {
-            MessageBoxResult confirm = MessageBox.Show("Do you want to create this client?", "Confirm", MessageBoxButton.YesNo, MessageBoxImage.Question);
-            if (confirm == MessageBoxResult.Yes)
+            var confirm = _messageBoxService.Confirm("Do you want to create this client?");
+            if (confirm == true)
             {
                 var newClient = new CreateClientModel
```

Comment:
:pick: In fact, it can be simplified from 

``` csharp
   var confirm = _messageBoxService.Confirm("Do you want to create this client?");
   if (confirm == true)
```

to

``` csharp
if (_messageBoxService.Confirm("Do you want to create this client?"))
```